visualizer/parseTree.js

```diff
@@ -662,6 +662,7 @@
   // Refresh the parse tree after attempting to parse the input.
   ohmEditor.addListener('parse:input', function(matchResult, trace) {
     $('#bottomSection .overlay').style.width = 0;  // Hide the overlay.
+    ohmEditor.parseTree.rootTrace = trace;
```

Comment:
Does the root trace need to be public? It seems like this only references to it are to pass it back in as an argument to `ohmEditor.parseTree.refresh`. If that's the case, then we can probably just get rid of that argument, and keep the root trace in a variable inside of parseTree.js.